src/beamlime/applications/groups.py

```diff
@@ -16,19 +15,34 @@
 # TODO: Implement multi-process/multi-machine instance group interface.
 class BeamlimeApplicationInstanceGroup(BeamlimeApplicationInterface):
     """
-    Multiple instances of a single type of application.
-    It overwrites all interfaces of ``BeamlimeApplicationInterface``,
-    in a way that it has the exact same interfaces
-    just like a single application but controls multiple instances internally.
+    Controls instance(s) of an application.
+    It overwrites all interfaces of ``BeamlimeApplicationInterface``.
+    It has the exact same interfaces as a single application instance,
+    but controls multiple instances internally.
     Therefore it does not inherit ``BeamlimeApplicationInterface``,
     but it should pass the protocol test.
     See ``protocol_test.test_instace_group_protocol`` of the unit test package.
 
     Parameters
     ----------
-    app_name: str
+    name: str
         Name of the application.
 
+    constructor: str
```

Comment:
We are using https://github.com/tox-dev/sphinx-autodoc-typehints, so there is no need to repeat the type in the docstring. The Sphinx plugin will take care of that. Not sure if this is configured for `beamlime` yet.